Data_Collection/CRON_scripts/get_data.py

```diff
@@ -103,7 +106,10 @@ def compute_clusters(curve, t, plot = False):
     
     # Computing the correlation matrix based on the maximum correlation lag 
     
-    del inc_canton['CHFL']
+    try:
+        del inc_canton['CHFL']
+    except:
+        import pdb; pdb.set_trace()
```

Comment:
This may not be a good Idea as it will stop the execution, right? Perhaps we should just log the exception, and continue.

Reply:
oh sure thing .. sorry I used that for debugging and forgot to remove that. thanks for caching that.